CIME/case/case.py

```diff
@@ -356,6 +358,9 @@ def read_xml(self):
         self._env_entryid_files.append(
             EnvWorkflow(self._caseroot, read_only=self._force_read_only)
         )
+        self._env_entryid_files.append(
+            EnvPostprocessing(self._caseroot, read_only=self._force_read_only)
```

Comment:
Here I think that you want to make sure POSTPROCESSING_SPEC_FILE exists before you create an EnvPostprocessing object.

Reply:
I mentioned this on Slack, but realized I should keep this thread going as well... `self.get_value("POSTPROCESSING_SPEC_FILE")` is `None` at this point. I tried other `env_case.xml` flags like `MODEL` and got `None` as well, so I think `get_values()` doesn't work. How can I get the value of `POSTPROCESSING_SPEC_FILE`?

Reply:
Maybe generate the almost empty env_postprocessing.xml here and then delete it when POSTPROCESSING_SPEC_FILE is available.

Reply:
We'll see how testing goes, but I think the logic for removing `env_postprocessing.xml` if it is empty actually belongs in `Case.flush()` instead of `Case.configure()`; that's what d393f0c5b does (though I realized I spun up a new sandbox on my laptop to test this off the supercomputer and didn't run `pre-commit`... so there will be at least one more "fix CI" commit coming)